.github/workflows/release.yml

```diff
@@ -0,0 +1,20 @@
+name: Release
+on:
+  push:
+    branches: [master]
+    tags: ["*"]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: olafurpg/setup-scala@v2
+      - uses: olafurpg/setup-gpg@v2
+      - run: git fetch --tags --unshallow
+      - name: Publish ${{ github.ref }}
+        run: sbt ci-release docs/docusaurusPublishGhpages
+        env:
```

Comment:
`scalameta` uses something very similar, a few additional variables: https://github.com/scalameta/scalameta/blob/master/.github/workflows/release.yml#L19

Reply:
Hm, I don't know what they can be used for. Website deploy may be?

Reply:
idk